### DISCUSSION TITLE
libvips meetup, London, Thursday 5 January 2023

### DISCUSSION THREAD

**Discussion by jcupitt** [libvips/libvips#3228] [General]
We're planning a libvips meetup at the Syrian Kitchen in central London on Thursday 5th January. Me, Kirk and Lovell are confirmed so far.  If you'd like a free meal and some dorky chat, please join us! 

https://www.imadssyriankitchen.co.uk

I'll book the table some time in late December, so you'll need to ask for a place before then.

John

**Comment by ljwagerfield** [comment]
I'd love to come to this ❤️ 

Please can you sign me up?

**Reply by jcupitt** [reply]
Sure, let's make a list of people who'd like to come in late dec, and book a table.

**Comment by jcupitt** [comment]
We have four confirmed (me, ljwagerfield, lovell, kirk) plus one probable (jpadfield), so I've booked for five at 8:30pm. If anyone else turns up, perhaps we can squeeze them in too.

It's just next to the Blue Posts, so we could perhaps meet there beforehand.

**Reply by lovell** [reply]
https://www.greeneking-pubs.co.uk/pubs/greater-london/blue-posts/

(as there are still at least 3 pubs nearby with the same name, and I've gone to the wrong one at least once before)

**Reply by jcupitt** [reply]
Ah good point!

**Reply by RReverser** [reply]
I'd love to come in, but still not sure whether I'll be able to make it. Does a tentative yes count?

**Reply by jcupitt** [reply]
We have a table for five and four confirmed, so it might work. And restaurants can often squeeze one more in if you beg.

The pub beforehand (and maybe after) would certainly be fine, of course.

**Reply by jcupitt** [reply]
... or if you can confirm, I'll phone the restaurant and ask for another place.

**Reply by RReverser** [reply]
Sorry, looks like won't make it after all :(

**Reply by jcupitt** [reply]
Ah shame! Maybe next year.

**Reply by kmartinez** [reply]
I assume the pub is this one? https://goo.gl/maps/M9y5Go1GGD4387Kf8

**Reply by jcupitt** [reply]
There are three blue posts pubs in Soho, one of them is next to the restaurant. Lovell posted a link above ^^

**Reply by jcupitt** [reply]
This one https://goo.gl/maps/Uf9KcSxfL3GqPh8R6

**Comment by jcupitt** [comment]
Kirk and I have arranged to meet at the Blue Posts at 7pm:

https://goo.gl/maps/Uf9KcSxfL3GqPh8R6

So you all there, or in the restaurant if you prefer.

**Comment by ljwagerfield** [comment]
Unfortunately, I wasn't aware there are [train strikes](https://www.nationalrail.co.uk/times_fares/147897.aspx) today 😢

Please accept my sincerest apologies for the late notice, I hope you all have a great evening!

**Reply by jcupitt** [reply]
Oh no! I'd forgotten about the strikes. Maybe next year.

**Reply by ljwagerfield** [reply]
Absolutely! I won't get a chance to say this in person, unfortunately, but thank you @jcupitt, @lovell, and everyone else who's involved in the development of libvips: the software industry is built on smart, generous persons like yourselves!

**Comment by jcupitt** [comment]
That was a lot of fun. Thank you everyone!

@lovell you mentioned sharp being in a top 10 web framework list somewhere. Do you remember where? I'd like to be able to boast to my mum about it heh.

**Reply by lovell** [reply]
Despite not being a web framework, it's in the list of "top web frameworks" selected by the Google Chrome team for funding - see https://developer.chrome.com/blog/framework-fund-2022/

**Comment by lovell** [comment]
Here's a photo from yesterday evening of Kirk and John discussing something very important.

![](https://user-images.githubusercontent.com/210965/211038178-fd5c6958-7917-4502-9bef-e671dab25b8e.jpg)

**Comment by jcupitt** [comment]
Argh I just tested +ve for covid. I might have been infectious when we met :( I hope you're all OK.

**Reply by lovell** [reply]
Oh no, hope you and the family have not been too badly affected. No symptoms here :crossed_fingers:

**Comment by lovell** [comment]
For those who haven't already seen, we're running another of these events in London next Wednesday - details at https://github.com/libvips/libvips/discussions/3748
